server/coordinator/procedure/create_drop_partition_table_test.go

```diff
@@ -21,7 +21,7 @@ func TestCreateAndDropPartitionTable(t *testing.T) {
 
 	shardPicker := NewRandomBalancedShardPicker(manager)
 
-	testTableNum := 20
+	testTableNum := 8
```

Comment:
Why change the number of tables?

Reply:
There is no need to create so many partition tables, the current value has already covered the required path.